axum/src/extract/path/mod.rs

```diff
@@ -25,6 +25,8 @@ use std::{
 ///
 /// # Example
 ///
+/// These examples assume the "serde" feature of the `uuid` crate is enabled.
```

Comment:
```suggestion
/// These examples assume the `serde` feature of the [`uuid`] crate is enabled.
///
/// [`uuid`]: https://crates.io/crates/uuid
```

Reply:
Ah, I did hesitate to do this. I guess I should have :smile: 

Thanks!